desc/plotting.py

```diff
@@ -434,7 +434,15 @@ def plot_1d(eq, name, grid=None, log=False, ax=None, **kwargs):
     if len(plot_axes) != 1:
         return ValueError(colored("Grid must be 1D", "red"))
 
-    data, label = _compute(eq, name, grid, kwargs.pop("component", None))
+    if (
+        eq.iota is None
```

Comment:
would it be cleaner to move this stuff to the `_compute` function?

Reply:
Ah, you are probably right